Mathlib/Algebra/Algebra/Operations.lean

```diff
@@ -355,11 +355,11 @@ theorem one_eq_range : (1 : Submodule R A) = LinearMap.range (Algebra.linearMap
     LinearMap.toSpanSingleton_eq_algebra_linearMap]
 
 theorem algebraMap_mem (r : R) : algebraMap R A r ∈ (1 : Submodule R A) := by
-  rw [one_eq_range]; exact LinearMap.mem_range_self _ _
+  simp [one_eq_range]
```

Comment:
Did you manually check if these simp's are quite slow? (Benchmarking also works, if you prefer.)

Reply:
On the order on 1000 heartbeats, half of that when squeezed. Not worth it, IMO.